.github/workflows/deploy-docker-tag.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     tags:
       - "v*"
+    paths:
```

Comment:
What if dependencies in the Gemfile change?

Reply:
Also, I think `package.json` and `package-lock.json` and `bin/entry_point.sh` should be included. I have to see if any other file exist for this CI/CD.

Reply:
Nice, that is exactly why I added you as a reviewer since you are our docker master haha

Reply:
If you find any other additions (or don't) let me know so I can merge these on master